README.md

```diff
@@ -1,7 +1,7 @@
 <!--
```

Comment:
Can you please generate README.md once more?
#2440 has been merged and a lot of documents with images are available now, so I think some new galleries will be created.

Reply:
there is no change because the max num of image is 15.